src/metrics/misc.ts

```diff
@@ -33,6 +33,14 @@ export class MetadataMetric implements Metric {
   blackWins: number;
   whiteWins: number;
   ties: number;
+  openings: 
+  { [opening: string]: 
+    { appearances: number, 
+      blackWins: number, 
+      whiteWins: number, 
+      whiteToBlackWinRatio: number }
+  };
+  bongcloud: number;
```

Comment:
we shouldn't need to track this separately right?

Reply:
yes but i wanted to do it just for u